lib/logstash/filters/preprocessor.rb

```diff
@@ -76,6 +84,18 @@ def regex_generator(logformat)
     Regexp.new("^#{format}$")
   end
 
+  def preprocess(log_line, regexes)
+    log_line = " #{log_line}"
+    regexes.each do |regex|
+      log_line = log_line.gsub(regex, '<*>')
+    end
+
+    @general_regex.each do |regex|
+      log_line = log_line.gsub(regex, '<*>')
+    end
+    log_line
+  end
```

Comment:
Comment this function please. Also can we rename the function to be more specific?